src/olympia/amo/tests/test_amo_utils.py

```diff
@@ -26,12 +25,14 @@
 u = u'Ελληνικά'
 
 
-def test_slug_validator():
-    eq_(slug_validator(u.lower()), None)
-    eq_(slug_validator('-'.join([u.lower(), u.lower()])), None)
-    assert_raises(ValidationError, slug_validator, '234.add')
-    assert_raises(ValidationError, slug_validator, 'a a a')
-    assert_raises(ValidationError, slug_validator, 'tags/')
+class TestSlugValidator(TestCase):
```

Comment:
I might be wrong but imho `TestCase` adds database functionality to the test-suite and we don't need that for this test so we could potentially skip inheriting from `TestCase` - we don't have to anyway with pytest. The `assertRaises` could become `pytest.raises(Exception, func, *args, **kwargs)`

but let's not inherit from `TestCase` if we don't have to, 'cause it'll add quite a bit of execution time and our test-suite already is slow as hell.